### ISSUE TITLE
[Bug]: Don't save space of the latest line

### ISSUE THREAD

**Issue by bassem-saeed** [AndroidIDEOfficial/AndroidIDE#991]
### What happened?

As an example if I at the 32 line and I wrote two tabs then when I click enter to make a new line the two tabs of the 32 line not show and I should make the two tabs again, Can you solve it?

### What's the expected behavior?

Solving the problem..

### What version of AndroidIDE you're using?

v2.4.0 (debug builds)

### Relevant log output

```shell
No output
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
Do you mean that the indentation is not continued to the next line? What file you're editing?

**Comment by bassem-saeed** [comment]
Yes, I edit the MainActivity file, and in any kotlin file the problem happen

**Comment by Smooth-E** [comment]
From your initial message it looks like you are adding extra indentation yourself, breaking the formatting rules. In such case any IDE would ignore your additional indentation.

If the default indentation size isn't good for you, you can change it in *IDE Preferences -> Editor -> Tab size*

**Comment by bassem-saeed** [comment]
Bro as an example if I clicked the tab twice then I clicked enter to go to the next line the double tab that I click at the last line don't show at the current line
